spells.txt

```diff
@@ -79,3 +81,12 @@
   sound = SPELL_BLAST
   message = "enter berserk mode" "goes berserk"
 }
+
+#GD ----------------------------------------------------------------------------------------
+
+"Explosive Overclock"
+{
+symbol = "X"
+cooldown = 1
+effect = Filter AUTOMATON Chain { Area 5 Chain{Damage SPELL_DAMAGE SPELL Blast} Area 3 PlaceFurniture "FIRE_WALL" Suicide DIE}
```

Comment:
this is a self-cast spell, please remove Filter AUTOMATON part

Reply:
Fixed